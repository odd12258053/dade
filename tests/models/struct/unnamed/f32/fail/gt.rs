use dade::model;
#[model]
struct TestModel (
   #[field(gt = 2)]
    f32
);
fn main() {}
