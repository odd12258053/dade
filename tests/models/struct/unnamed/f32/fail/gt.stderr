error[E0277]: can't compare `f32` with `{integer}`
 --> tests/models/struct/unnamed/f32/fail/gt.rs:2:1
  |
2 | #[model]
  | ^^^^^^^^ no implementation for `f32 < {integer}` and `f32 > {integer}`
  |
  = help: the trait `PartialOrd<{integer}>` is not implemented for `f32`
  = help: the following other types implement trait `PartialOrd<Rhs>`:
            f32
            f64
            i128
            i16
            i32
            i64
            i8
            isize
          and $N others
  = note: this error originates in the attribute macro `model` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider using a floating-point literal by writing it with `.0`
  |
4 |    #[field(gt = 2.0)]
  |                  ++
